.redhat-ci.yaml

```diff
@@ -67,7 +18,6 @@ host:
 branches:
     - master
```

Comment:
SImilarly for `pulls: true`. Also, the `required` key is only useful if you're using a merge bot. Did you also want homu hooked up to this repo?